cmd/aro/operator.go

```diff
@@ -87,10 +87,11 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 	if err != nil {
 		return err
 	}
-	arocli, err := aroclient.NewForConfig(restConfig)
+	securitycli, err := securityclient.NewForConfig(restConfig)
 	if err != nil {
 		return err
 	}
+	// TODO (NE): dh is sometimes passed, sometimes created later. Can we standardize?
```

Comment:
There are a couple places where `dh` is created later:
- [pkg/operator/controllers/genevalogging/genevalogging_controller.go in `Reconcile` function](https://github.com/Azure/ARO-RP/blob/e4ff2a4328ccf77aa600d332364ae8fd4f7b28d9/pkg/operator/controllers/genevalogging/genevalogging_controller.go#L65-L72), with comment that it "currently saves us in the case that the controller runs before the SCC API is registered."
- [pkg/operator/controllers/routefix/routefix_controller.go in `deploy` function](https://github.com/Azure/ARO-RP/blob/e4ff2a4328ccf77aa600d332364ae8fd4f7b28d9/pkg/operator/controllers/routefix/routefix_controller.go#L86-L93), with comment that it "currently saves us in the case that the controller runs before the SCC API is registered."
- [pkg/operator/controllers/workaround/workaround_controller.go in `NewReconciler` function](https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/workaround/workaround_controller.go#L38-L41). No comment indicating why.

In these places, if we ever really had to create `dh` later (might have been copy/paste or just inconsistency), do we still have to?

Reply:
Or, conversely, should we always create `dh` at the lowest scope possible?